src/KubeLibrary/KubeLibrary.py

```diff
@@ -304,6 +304,14 @@ def filter_pods_containers_by_name(self, pods, name_pattern):
                     containers.append(container)
         return containers
 
+    def filter_service_names(self, service):
```

Comment:
I believe it should be filter_configmap... We should probably think of some generic way of filtering by name in future. filter_pods_names and filter_service_accounts_names are very similar.

Reply:
Done.